spam/02_spam_data_augmentation_tutorial.ipynb

```diff
@@ -2281,23 +2223,23 @@
      "output_type": "stream",
      "text": [
       "Restoring model weights from the end of the best epoch.\n",
-      "Epoch 00024: early stopping\n"
+      "Epoch 00017: early stopping\n"
      ]
     },
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
       "Restoring model weights from the end of the best epoch.\n",
-      "Epoch 00017: early stopping\n"
+      "Epoch 00020: early stopping\n"
      ]
     },
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Test Accuracy when training on original dataset: 0.932\n",
-      "Test Accuracy when training on augmented dataset: 0.928\n"
+      "Test Accuracy when training on original dataset: 0.916\n",
```

Comment:
looks like we need more reproducible results in keras...you can copy the block from the LF notebook

Reply:
Nice!